src/opentelemetry/launcher/configuration.py

```diff
@@ -94,6 +94,7 @@ def configure_opentelemetry(
     log_level: str = _OTEL_LOG_LEVEL,
     span_exporter_insecure: bool = _OTEL_EXPORTER_OTLP_SPAN_INSECURE,
     metric_exporter_insecure: bool = (_OTEL_EXPORTER_OTLP_METRIC_INSECURE),
+    auto_instrumented: bool = False,
```

Comment:
does Python have a thing where we can put an _ in front of this to make it private?

Reply:
it does, I didn't realize this would work for method arguments, good suggestion!

Reply:
I also think using private arguments is the best way to go here. Just for the record, we can use function attributes if we don't want to add another argument, but for this situation, it is not so "clean" as the variables that affect the call are not in the call themselves:

```python
def configure_opentelemetry():
    if hasattr(configure_opentelemetry, "_auto_instrumented"):
        print("auto instrumented")
    else:
        print("not auto instrumented")

configure_opentelemetry()

configure_opentelemetry._auto_instrumented = True

configure_opentelemetry()
```
```
not auto instrumented
auto instrumented
```

Reply:
oo I learned a lot here - did not know about those function attributes!!  Glad it worked to make this private